pkgs/servers/home-assistant/custom-components/adaptive-lighting/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub
+, buildHomeAssistantComponent
+, home-assistant
```

```suggestion
, ulid-transform
```

Consume the python dependency directly.